src/components/SidePane/Help/index.tsx

```diff
@@ -1,25 +1,79 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 // eslint-disable-next-line no-restricted-imports
-import {Animated} from 'react-native';
+import {Animated, View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+// @ts-expect-error This is a workaround to display HelpPane on top of everything,
+// Modal from react-native can't be used here, as it would block interactions with the rest of the app
+import ModalPortal from 'react-native-web/dist/exports/Modal/ModalPortal';
+import FocusTrapForModal from '@components/FocusTrap/FocusTrapForModal';
+import SidePaneOverlay from '@components/SidePane/SidePaneOverlay';
 import useKeyboardShortcut from '@hooks/useKeyboardShortcut';
 import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useSafeAreaPaddings from '@hooks/useSafeAreaPaddings';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
 import HelpContent from './HelpContent';
 import type HelpProps from './types';
 
-function Help({sidePaneTranslateX, closeSidePane}: HelpProps) {
+function Help({sidePaneTranslateX, closeSidePane, shouldHideSidePaneBackdrop}: HelpProps) {
     const styles = useThemeStyles();
     const {isExtraLargeScreenWidth, shouldUseNarrowLayout} = useResponsiveLayout();
     const {paddingTop, paddingBottom} = useSafeAreaPaddings();
+    const [isRHPVisible = false] = useOnyx(ONYXKEYS.MODAL, {selector: (modal) => modal?.type === CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED});
```

Comment:
This logic is necessary to prevent showing overlay twice:

https://github.com/user-attachments/assets/6d3f2166-fab2-4f68-8969-8a6da9c41b35